pages/FAQ/_index.md

```diff
@@ -125,6 +111,8 @@ Use a tool like for example `lxappearance` to change the GTK cursor.
 After that, add `exec-once=hyprctl setcursor [THEME] [SIZE]` to your config and
 restart Hyprland.
 
+If you want to change cursor size on QT, add `XCURSOR_SIZE=[SIZE]` to your wrapper.
```

Comment:
not needed, Hyprland exports to 24 now

Reply:
just mention default 24 then

Reply:
```diff
-If you want to change cursor size on QT, add `XCURSOR_SIZE=[SIZE]` to your wrapper.
+ As of [commit/release] Hyprland exports `XCURSOR_SIZE`, 24 being the default. You may export a different value in your wrapper to change it .
```

How does this sound?

Reply:
No need to mention the commit release. We have a versioned wiki.

Reply:
True. So just *Hyprland exports `XCURSOR_SIZE`, 24 being the default...*?

Reply:
ye